<ui version="4.0" >
 <class>StudyConfigurationDialog</class>
 <widget class="QDialog" name="StudyConfigurationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Study Configuration</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTreeView" name="treeView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>9</number>
     </property>
     <widget class="QWidget" name="ecgPage" />
     <widget class="QWidget" name="pressurePage" />
     <widget class="QWidget" name="catheterBlockAPage" />
     <widget class="QWidget" name="catheterBlockBPage" />
     <widget class="QWidget" name="catheterBlockCPage" />
     <widget class="QWidget" name="catheterBlockDPage" />
     <widget class="QWidget" name="catheterBlockEPage" />
     <widget class="QWidget" name="catheterBlockFPage" />
     <widget class="QWidget" name="catheterBlockGPage" />
     <widget class="QWidget" name="stimPage" />
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="saveButton" >
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="saveAsButton" >
     <property name="text" >
      <string>Save As...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
